repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,15 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
```

Comment:
can we have this defined in the config.json as well, along with a description of it's purpose